src/modules/profile/components/ProfileList/ProfileListItemClear.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import { ProfileDataItem } from "../../mock/profileData";
+import ImagePicker from "../ImagePicker/ImagePicker";
+
+interface Props {
+    item: ProfileDataItem;
+    inputHandler: (key: string) => (value: string) => void,
+    value?: any
+}
+
+const  getInputComponent = (props: Props) => {
+    const {item, inputHandler, value} = props;
+    switch (item.input.type) {
+        default:
+            return <ImagePicker imageHandler={inputHandler(item.key)} value={value} />
+    }
```

Comment:
What's the point? You have only one case, just put it directly to the `ProfileListItemClear`

Reply:
Read about YAGNI